plugin/pkg/auth/authorizer/rbac/bootstrappolicy/policy.go

```diff
@@ -279,8 +279,18 @@ func ClusterRoles() []rbac.ClusterRole {
 				rbac.NewRule("get").Groups(legacyGroup).Resources("endpoints", "namespaces", "serviceaccounts").RuleOrDie(),
 				rbac.NewRule("update").Groups(legacyGroup).Resources("endpoints", "serviceaccounts").RuleOrDie(),
 
-				rbac.NewRule("list", "watch").Groups("*").Resources("namespaces", "nodes", "persistentvolumeclaims",
-					"persistentvolumes", "pods", "secrets", "services", "serviceaccounts", "replicationcontrollers").RuleOrDie(),
+				rbac.NewRule("list", "watch").Groups("*").Resources(
```

Comment:
Looks like a bug crept in.  The groups should be called out individually so that you don't have access to `namespaces.extensions` (even though nothing is there).